src/shared/Core/Trace2.cs

```diff
@@ -796,3 +840,42 @@ protected override string GetEventMessage(Trace2FormatTarget formatTarget)
         return sb.ToString();
     }
 }
+
+public class ErrorMessage : Trace2Message
+{
+    [JsonProperty("msg", Order = 7)]
+    public string Message { get; set; }
+
+    [JsonProperty("format", Order = 8)]
+    public string FormattedErrorMessage { get; set; }
```

Comment:
The `FormattedErrorMessage` property name is a bit misleading, I think.  In the Trace2 spec, I have "msg" as the actual error message to be printed and "fmt" as the format string used in a `sprintf()` like call to build "msg".  So would it be better to call this `FormatString` or `ParameterizedMessage` or something??

Reply:
Sure, I like `ParameterizedMessage`. That'll be an easy change to make.